lms/static/sass/discussion/utilities/_variables.scss

```diff
@@ -6,3 +10,8 @@ $forum-color-following: $blue;
 $forum-color-staff: $blue;
 $forum-color-community-ta: $green-d1;
 $forum-color-marked-answer: $green-d1;
+
+// post images
+$post-image-dimension: ($baseline*3);  // image size  + margin
```

Comment:
> The benefit of that is we could show all image sizes in the platform in 1 place, and reuse sizes when appropriate. (profile-image, header-image, discussion-post, discussion-response, etc.)

I like that idea a lot (canned profile image sizes + specific overrides per-app that we control). If its not too much work, great, but if it is or you want to do that in a separate PR aside from this, I'm very much cool with that too.